src/storage/migration/migration_structures.rs

```diff
@@ -284,3 +287,202 @@ impl From<TlsBeforeV21> for TlsFromV21 {
+    pub src_as: u16,
+    pub dst_as: u16,
+    pub src_mask: u8,
+    pub dst_mask: u8,
+    pub sequence: u32,
+    pub engine_type: u8,
+    pub engine_id: u8,
+    pub sampling_mode: u8,
+    pub sampling_rate: u16,
+}
+
+impl EventFilter for Netflow5BeforeV23 {
+    fn data_type(&self) -> String {
+        "netflow v5".to_string()
+    }
+    fn orig_addr(&self) -> Option<IpAddr> {
+        Some(self.src_addr)
+    }
+    fn resp_addr(&self) -> Option<IpAddr> {
+        Some(self.dst_addr)
+    }
+    fn orig_port(&self) -> Option<u16> {
+        Some(self.src_port)
+    }
+    fn resp_port(&self) -> Option<u16> {
+        Some(self.dst_port)
+    }
+    fn log_level(&self) -> Option<String> {
+        None
+    }
+    fn log_contents(&self) -> Option<String> {
+        None
+    }
+    fn sensor(&self) -> Option<String> {
+        Some(self.source.clone())
+    }
+}
+
+impl From<Netflow5BeforeV23> for crate::storage::Netflow5 {
+    fn from(input: Netflow5BeforeV23) -> Self {
+        Self {
+            src_addr: input.src_addr,
+            dst_addr: input.dst_addr,
+            next_hop: input.next_hop,
+            input: input.input,
+            output: input.output,
+            d_pkts: input.d_pkts,
+            d_octets: input.d_octets,
+            first: input.first,
+            last: input.last,
+            src_port: input.src_port,
+            dst_port: input.dst_port,
+            tcp_flags: input.tcp_flags,
+            prot: input.prot,
+            tos: input.tos,
+            src_as: input.src_as,
+            dst_as: input.dst_as,
+            src_mask: input.src_mask,
+            dst_mask: input.dst_mask,
+            sequence: input.sequence,
+            engine_type: input.engine_type,
+            engine_id: input.engine_id,
+            sampling_mode: input.sampling_mode,
+            sampling_rate: input.sampling_rate,
+        }
+    }
+}
+
+#[derive(Deserialize, Serialize)]
+pub struct Netflow9BeforeV23 {
+    pub source: String,
+    pub sequence: u32,
+    pub source_id: u32,
+    pub template_id: u16,
+    pub orig_addr: IpAddr,
+    pub orig_port: u16,
+    pub resp_addr: IpAddr,
+    pub resp_port: u16,
+    pub proto: u8,
+    pub contents: String,
+}
+
+impl EventFilter for Netflow9BeforeV23 {
+    fn data_type(&self) -> String {
+        "netflow v9".to_string()
+    }
+    fn orig_addr(&self) -> Option<IpAddr> {
+        Some(self.orig_addr)
+    }
+    fn resp_addr(&self) -> Option<IpAddr> {
+        Some(self.resp_addr)
+    }
+    fn orig_port(&self) -> Option<u16> {
+        Some(self.orig_port)
+    }
+    fn resp_port(&self) -> Option<u16> {
+        Some(self.resp_port)
+    }
+    fn log_level(&self) -> Option<String> {
+        None
+    }
+    fn log_contents(&self) -> Option<String> {
+        Some(self.contents.clone())
+    }
+    fn sensor(&self) -> Option<String> {
+        Some(self.source.clone())
+    }
+}
+
+impl From<Netflow9BeforeV23> for crate::storage::Netflow9 {
```

Comment:
It would be nice to unify how structure names are used within `migration_structures`. 
(rename to alias or use existing structure name).

Reply:
Sounds good, done.